docs/developer.md

```diff
@@ -79,3 +79,8 @@ hamlet.model is a copy of all_theses_no_split_w4_s52.model. This is a model trai
 `hamlet/testmodels/` contains some smaller models not suitable for production, but usable for testing (and small enough to be pushed to GitHub, although it will complain, and hence used on Travis). You can configure your local settings to point at these files and that will suffice for development.
 
 These models don't represent the entire MIT thesis collection (that's what lets them be smaller), so don't be surprised if documents of interest are not present.
+
+`hamlet.settings.local` defaults to using the test model, since it is checked
+into version control. If you have a different model you want to use:
+* put it in `hamlet/model/hamlet.model` (along with any other numpy files it needs to work);
+* set `DJANGO_USE_LIVE_MODEL=True` in `.env`.
```

Comment:
What if we added a new setting called something like HAMLET_MODEL that specified the path to an alternate model? This would have the advantage of being more flexible in where it lives and what it's called, as well as being more explicit about what's getting used. If the setting doesn't exist we can fall back to the testmodel.

Reply:
Presumed to be the relative path from the project root? Sure, that works. Will do.